src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/url-manager.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+
+    'use strict';
+
+    define(['app-config'], function(AppConfig) {
+
+        return {
+
+            name: 'url-manager',
+
+            initialize: function(app) {
+                var sandbox = app.sandbox,
+                    urlStore = {};
+
+                sandbox.urlManager = {};
+
+                /**
+                 * Set a url in the url store
+                 * @method setUrl
+                 * @param {String} key
+                 * @param {Mixed} urlTpl
+                 * @param {Function} handler
+                 */
+                sandbox.urlManager.setUrl = function(key, urlTpl, handler) {
+                    urlStore[key] = {
+                        template: urlTpl,
```

Comment:
We are not using abbreviations, so it should be `urlTemplate`, or maybe we can even just call it `template`?